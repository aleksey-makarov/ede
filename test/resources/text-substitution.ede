{ "single": "Lorem ipsum-Phrase" }
---
{{ single | takeWord }}
{{ single | dropWord }}
{{ single | dropUpper }}
{{ single | toCamel| dropLower }}
{{ single | stripPrefix("Lorem ") }}
{{ single | stripSuffix("Phrase") }}
{{ single | replace("ipsum-", "Hi! ") }}
{{ single | remove(" ipsum-") }}
{{ single | remove("e") }}
