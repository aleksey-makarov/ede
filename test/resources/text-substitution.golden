Lorem
ipsum-Phrase
orem ipsum-Phrase
IpsumPhrase
ipsum-Phrase
Lorem ipsum-
Hi! 
LoremPhrase
Lorm ipsum-Phras
